lib/phoenix_live_view/upload_config.ex

```diff
@@ -507,13 +507,7 @@ defmodule Phoenix.LiveView.UploadConfig do
     if too_many_files?(new_conf) do
       {:error, put_error(new_conf, new_conf.ref, @too_many_files)}
     else
-      case new_conf do
-        %UploadConfig{errors: []} = new_conf ->
-          {:ok, new_conf}
-
-        %UploadConfig{errors: [_ | _]} = new_conf ->
-          {:error, new_conf}
-      end
+      {:ok, new_conf }
```

Comment:
Unfortunately change this return type makes it so we generate preflights to Amazon even if there are errors. Perhaps a better solution is to this bit in channel.ex:

```
        new_upload_names =
          case ok_or_error do
            :ok -> Map.put(state.upload_names, conf.name, {upload_ref, cid})
            _ -> state.upload_names
          end
```

So we store the new upload names, even if it fails? Although I don't fully understand the impact of that.